molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -98,14 +98,16 @@
 
     - name: Create docker network(s)
       docker_network:
-        name: "{{ item }}"
+        name: "{{ item.name }}"
+        ipam_config: "{{ item.ipam_config | default(omit) }}"
+        ipam_options: "{{ item.ipam_options | default(omit) }}"
         docker_host: "{{ item.docker_host | default(lookup('env', 'DOCKER_HOST') or 'unix://var/run/docker.sock') }}"
         cacert_path: "{{ item.cacert_path | default((lookup('env', 'DOCKER_CERT_PATH') + '/ca.pem') if lookup('env', 'DOCKER_CERT_PATH') else omit) }}"
         cert_path: "{{ item.cert_path | default((lookup('env', 'DOCKER_CERT_PATH') + '/cert.pem')  if lookup('env', 'DOCKER_CERT_PATH') else omit) }}"
         key_path: "{{ item.key_path | default((lookup('env', 'DOCKER_CERT_PATH') + '/key.pem') if lookup('env', 'DOCKER_CERT_PATH') else omit) }}"
         tls_verify: "{{ item.tls_verify | default(lookup('env', 'DOCKER_TLS_VERIFY')) or false }}"
         state: present
-      with_items: "{{ molecule_yml.platforms | molecule_get_docker_networks }}"
+      with_items: "{{ molecule_yml.platforms | molecule_get_docker_network_configs }}"
```

Comment:
> This also introduces two new filters so we do not break creation in case create.yml was customized.

This is good but also leaves custom playbooks behind (although they are probably already struggling for other reasons). So, we'll need to include a changelog entry and documentation to announce this new stuff.